website/pages/docs/deployment/airflow.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Orchestrating CloudQuery Syncs with Airflow
```

Comment:
I think it should be clear from the title that it is Airflow on K8s rather than just plain Airflow. The distinction is important because AWS and GCP both have managed airflow services

Reply:
Good point, updated. I hope in the future we can add a few more examples (maybe on a different page) about deploying to the managed AWS / GCP versions or deploying using the BashOperator.